include/avif/internal.h

```diff
@@ -450,6 +462,10 @@ AVIF_NODISCARD avifBool avifAreGridDimensionsValid(avifPixelFormat yuvFormat,
 // image->imir on success. Returns AVIF_RESULT_INVALID_EXIF_PAYLOAD on failure.
 avifResult avifImageExtractExifOrientationToIrotImir(avifImage * image);
 
+// Returns the Exif orientation in [1-8] as defined in JEITA CP-3451C section 4.6.4.A Orientation
+// corresponding to image->irot and image->imir.
+uint8_t avifImageIrotImirToExifOrientation(const avifImage * image);
```

Comment:
This function is only used in src/write.c now. Move this function to src/write.c as a static function, or put this function inside `#if defined(AVIF_ENABLE_EXPERIMENTAL_METAV1)`.